app/lib/screens/EditorScreen/Editor.dart

```diff
@@ -0,0 +1,355 @@
+                        itemBuilder: (context, index) {
+                          return TagListItem(
+                            tagTitle: _tagList[index],
+                            tagColor: _tagColorList[index],
+                          );
+                        },
+                      ),
+                    )
+                  else
+                    Container()
+                ],
+              ),
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+
+  void showBottomSheet() {
+    Color? primaryColor = Colors.grey;
+
+    showModalBottomSheet(
+      context: context,
+      builder: (context) {
+        return Column(
+          children: [
+            const Padding(
+              padding: EdgeInsets.all(12.0),
+              child: Align(
+                alignment: Alignment.center,
+                child: Text(
+                  'Edit Tag',
+                  style: TextStyle(fontSize: 16),
+                ),
+              ),
+            ),
+            Container(
+              padding: const EdgeInsets.all(10),
+              child: Container(
+                width: double.infinity,
+                height: 40,
+                decoration: BoxDecoration(
+                  color: AppColors.textFieldColor,
+                  borderRadius: BorderRadius.circular(15),
+                ),
+                child: TextField(
+                  style: TextStyle(color: primaryColor),
+                  controller: tagTitleController,
+                  cursorColor: Colors.white,
+                  textAlign: TextAlign.center,
+                  decoration: InputDecoration(
+                    hintText: 'Tag Name',
+                    hintStyle: TextStyle(
+                      color: Colors.white.withOpacity(0.5),
+                    ),
+                    border: InputBorder.none,
+                  ),
+                ),
+              ),
+            ),
+            const Padding(
+              padding: EdgeInsets.all(12.0),
+              child: Align(
+                alignment: Alignment.center,
+                child: Text(
+                  'Select a color',
+                  style: TextStyle(fontSize: 16),
+                ),
+              ),
+            ),
+            Expanded(
+              child: GridView.count(
+                crossAxisCount: 4,
+                children: List.generate(8, (index) {
+                  return Center(
+                    child: Padding(
+                      padding: const EdgeInsets.only(left: 8.0, right: 8.0),
+                      child: ElevatedButton(
+                        onPressed: () {
+                          setState(() {
+                            tagColor = arr[index];
+                            print(tagColor?.value.toString());
+                          });
+                        },
+                        child: const Text(''),
+                        style: ElevatedButton.styleFrom(primary: arr[index]),
```

Comment:
Upon selection the inlined tag now shows the color, but I can't see which one I selected in this view though. There must still be something missing here.